sources/Licenses/Licenses.yml

```
@@ -0,0 +1,53 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  # Apache[ -]?2(?:\.0)?: Apache 2.0 # See notes, will be added later.
+  "[0O]BSD": 0BSD
```

https://github.com/search?q=language%3Amarkdown+%220-bsd%22&type=code

There are some 0-BSD

So this should he OK

```suggestion
  "[0O]\-?BSD": 0BSD
```

But, we have to be careful about "0 BSD"

Because we have some "apache 2.0 BSD" text, this one should be added as valid one

https://github.com/search?q=language%3Amarkdown+%220+bsd%22&type=code

Same for "o BSD" because of Brazilian

https://github.com/search?q=language%3Amarkdown+%22+O+bsd%22&type=code

---

Thanks for the explanation. I appreciate your attention to detail.